build.gradle.kts

```
@@ -0,0 +1,57 @@
+plugins {
+    id("java")
+    id("io.github.goooler.shadow") version "8.1.7"
+}
+
+group = "re.imc"
+version = "1.0.0"
+
+repositories {
+    mavenCentral()
+    maven("https://repo.papermc.io/repository/maven-public/")
+    maven("https://central.sonatype.com/repository/maven-snapshots/")
+
+    maven("https://mvn.lumine.io/repository/maven-public/")
+
+    maven("https://repo.opencollab.dev/maven-releases/")
+    maven("https://repo.opencollab.dev/maven-snapshots/")
```

```suggestion
    maven("https://repo.opencollab.dev/main/")
```
covers both snapshot and release